bin/lms-maketcnew.php

```diff
@@ -471,26 +513,28 @@ function ($value) {
     $upceil_n = (!$channel['upceil_n'] ? $uprate_n : $channel['upceil_n']);
     $downrate_n = $channel['downrate_n'];
     $downceil_n = (!$channel['downceil_n'] ? $downrate_n : $channel['downceil_n']);
+    $date_readable = date('YYYY-mm-dd HH:MM', time());
```

Comment:
a może:
```php
$date_hf = .... // Human Friendly
$date_uts = ... / Unix TimeStamp
```
?

Reply:
W sumie to nawet zamiast date_hf można dać po prostu date, a date_uts bez zmian, albo w ogóle zamiast date_uts po prostu uts.

Reply:
https://github.com/chilek/lms/pull/2314/commits/976419c818694b8e2871ad088df438ef311ae528